catalog/transformer/catalog-transformer-streaming-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -74,13 +74,27 @@
     </cm:managed-service-factory>
 
 
-    <reference id="gml3ToWktParser" interface="org.codice.ddf.parser.Parser"/>
+    <bean id="gml3Configuration" class="org.geotools.gml3.GMLConfiguration"/>
+    <bean id="gml3_2Configuration" class="org.geotools.gml3.v3_2.GMLConfiguration"/>
```

Comment:
I haven't seen an `_` convention before on the bean id.

Reply:
Changed to `v32`